test_harness/reps.go

```diff
@@ -15,12 +15,11 @@ type Scenario struct {
 
 type TestCase struct {
 	ExpectedValue interface{}   `json:"expectedValue"`
-	ExpectError   bool          `json:"expectError"`
 	User          ldclient.User `json:"user"`
 }
 
 type EvalFeatureRequest struct {
-	Type         string        `json:"type"`
+	ValueType    string        `json:"valueType"`
```

Comment:
I changed the name of this field to be more friendly to languages that have 'value' as a reserved word.